models/target.go

```diff
@@ -83,6 +83,7 @@ type VT struct {
 type GetVT struct {
 	cmds.EventType
 	messages.Identifier
+	OID string `json:"oid"`
```

Comment:
why are not using the normal get command (ID instead of OID)? That way it would be easier to use and not require an update on the library; especially since OID is essentially an ID I don't understand the differentiation.

Reply:
Maybe it is too overcomplicated as different sensor should not have different feed states

Reply:
Yes, especially since you would need to get a scan id to get vt information which isn't always possible from a client point of view.